src/SDAO.t.sol

```diff
@@ -475,6 +481,88 @@ contract SDAOTest is DssTest {
         vm.startPrank(sender);
         checkModifier(address(token), "SDAO/not-authorized", authedMethods);
     }
+
+    function testFile() public {
+        checkFileString(address(token), "SDAO", ["name"]);
+        checkFileString(address(token), "SDAO", ["symbol"]);
+    }
+
+    // There are no checkFileString on DssTest, so we need to implement it here.
```

Comment:
Non blocking for the audit, but would be nice if you can create a PR to dss-test repo with this.

Reply:
Will do.